parts/language/lang_en.lua

```diff
@@ -652,22 +652,22 @@ return{
             music="BGMs",
         },
         login={
-            title="Log In",
-            register="Register",
-            email="E-mail Address",
+            title="Sign In",
+            register="Sign Up",
+            email="Email Address",
             password="Password",
             keepPW="Remember me",
             login="Log In",
         },
         register={
-            title="Register",
-            login="Log In",
+            title="Sign Up",
+            login="Sign In",
             username="Username",
             email="Email Address",
             password="Password",
             password2="Re-enter Password",
-            register="Register",
-            registering="Waiting for response...",
+            register="Sign Up",
+            registering="Waiting for response…",
```

Comment:
Let's see how the font renders these characters... and what's the style standard. (In a Chinese font, the three-dot-as-one-character displays in the middle of the line, instead of aligned to the bottom. I think English writing prefers the dots on the bottom?)